tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/AWS/LambdaCommon.cs

```diff
@@ -156,5 +154,19 @@ internal static void Flush()
                 Serverless.Error("Could not flush to the extension", ex);
             }
         }
+
+        internal static string SerializeObject<T>(T obj)
```

Comment:
This should be private. Which makes me realize that a bunch of method in that class could also be private

Reply:
fixed this, and made all functions only accessed within the class private